collections/_contributors/Ruth-Ikegah.md

```diff
@@ -0,0 +1,9 @@
+---

```

Comment:
@usamaidrsk, will you lowercase all the letters in this filename? The uppercase letters cause this profile to be listed out of the existing alphabetical order.